modules/ROOT/content-nav.adoc

```diff
@@ -433,6 +433,7 @@
 **** xref:errors/gql-errors/52N24.adoc[]
 **** xref:errors/gql-errors/52N25.adoc[]
 **** xref:errors/gql-errors/52N26.adoc[]
+**** xref:errors/gql-errors/52N27.adoc[]
```

Comment:
Good catch. I actually found out another problem as well. I'll fix it in another PR.